lib/phoenix/live_dashboard/live/processes_live.ex

```diff
@@ -35,6 +35,16 @@ defmodule Phoenix.LiveDashboard.ProcessesLive do
     <div class="processes-page">
       <h5 class="card-title">Processes</h5>
 
+      <div class="processes-search">
+        <form phx-submit="search" class="form-inline">
```

Comment:
Maybe we can do phx-change with phx-debounce instead? Then we don't need to explicitly submit it.